packages/app/src/elements/item-view.ts

```diff
@@ -508,11 +513,15 @@ export class ItemView extends Routing(StateMixin(LitElement)) {
         if (!this._nameInput.reportValidity()) {
             return;
         }
+        const oneMonthAgo = sub(new Date(), { months: 1 });
         app.updateItem(this._item!, {
```

Comment:
Is there a way to _unset_ something here? I couldn't find it, so I just set `lastAudited` to a date far in the past, which will also work.

Reply:
Have you tried `{ lastAudited: undefined }`? Otherwise, `{ lastAudited: new Date(0) }` should also work.